src/Route/Parsers.elm

```diff
@@ -26,11 +26,21 @@ fqn : Parser FQN
 fqn =
     let
         segment =
+            Parser.oneOf
+                -- Special case ;. which is an escaped . (dot), since we also use
+                -- ';' as the separator character between namespace FQNs and
+                -- definition FQNs. (';' is not a valid character in FQNs and is
+                -- safe as a separator/escape character).
+                [ b (succeed (identity ".") |. s ";.")
```

Comment:
Having this special case here didn't feel great, but was the simplest when chomping deals with character to character which would lead to a clash with `/;/` (the namespace/definition separator).